exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -44,6 +46,11 @@
 ExportResultT = TypeVar("ExportResultT")
 
 
+class GRPCCompression(enum.Enum):
+    DEFAULT = 0
```

Comment:
I think this would be more clear to users

```suggestion
    NO_COMPRESSION = 0
```